src/MizarWidgets/SamplingWithFrameTrackWidgetTest.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#include <qnamespace.h>
+
+#include <QApplication>
+#include <QCheckBox>
+#include <QLabel>
+#include <memory>
+
+#include "MizarWidgets/SamplingWithFrameTrackWidget.h"
+
+using ::testing::NotNull;
+
+namespace orbit_mizar_widgets {
+
+constexpr const char* kOrgName = "The Orbit Authors";
+
+constexpr std::string_view kMultiplicityCorrectionEnabledLabel =
+    "Probability of false-alarm for at least one function:";
+constexpr std::string_view kMultiplicityCorrectionDisabledLabel =
+    "Probability of false-alarm for an individual function:";
+
+class SamplingWithFrameTrackWidgetTest : public ::testing::Test {
+ public:
+  SamplingWithFrameTrackWidgetTest() : widget_(std::make_unique<SamplingWithFrameTrackWidget>()) {
+    QApplication::setOrganizationName(kOrgName);
+    QApplication::setApplicationName("SamplingWithFrameTrackInputWidgetTest.InitIsCorrect");
```

Comment:
I do not want to discourage you to add this, but its only strictly necessary when `QSettings` is involved (and I dont think it is). 

Also `InitIsCorrect` ?

Just fyi: I found a neat way to set these automatically, aka in the test fixture, but with individual names for each test. From RetrieveInstancesWidgetTest.cpp:
```
constexpr const char* kOrganizationName = "The Orbit Authors";
constexpr const char* kApplicationName{"RetrieveInstancesWidgetTest"};

...

void SetUp() override {
    QCoreApplication::setOrganizationName(kOrganizationName);
    QString test_name = QString("%1 - %2").arg(
        kApplicationName, testing::UnitTest::GetInstance()->current_test_info()->name());
    QCoreApplication::setApplicationName(test_name);
  }
```

Reply:
If that's not necessary, I guess, I'll just remove it. But thank you for the nice suggestion.